proto/lbm/collection/v1/tx.proto

```diff
@@ -466,7 +466,7 @@ message MsgModify {
   // Note: if token type is of FTs, the index cannot be empty.
   string token_index = 4;
   // changes to apply.
-  // possible attribute keys on modifying collection: name, base_img_uri, meta.
+  // possible attribute keys on modifying collection: name, uri, base_img_uri (deprecated), meta.
```

Comment:
I think we don't need `base_image_uri` for now. Isn't it?

Reply:
The client didn't mention about tx. So I have no idea whether we can remove it.